### PR TITLE
Add missing oauth scopes on users and organization api

### PR DESCRIPTION
**Pull request by lucasvanhalst**

### CODE REVIEW

File: identityservice/organization/organizations_if.go [hunk 1]
```diff
@@ -101,8 +101,8 @@ func OrganizationsInterfaceRoutes(r *mux.Router, i OrganizationsInterface) {
...
-	r.Handle("/organizations/{globalid}/dns/{dnsname}", alice.New(newOauth2oauth_2_0Middleware([]string{}).Handler).Then(http.HandlerFunc(i.DeleteDns))).Methods("DELETE")
-	r.Handle("/organizations/{globalid}/tree", alice.New(newOauth2oauth_2_0Middleware([]string{}).Handler).Then(http.HandlerFunc(i.GetOrganizationTree))).Methods("GET")
+	r.Handle("/organizations/{globalid}/dns/{dnsname}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.CreateDns))).Methods("POST")
+	r.Handle("/organizations/{globalid}/dns/{dnsname}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.UpdateDns))).Methods("PUT")
+	r.Handle("/organizations/{globalid}/dns/{dnsname}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.DeleteDns))).Methods("DELETE")
+	r.Handle("/organizations/{globalid}/tree", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:member", "organization:owner"}).Handler).Then(http.HandlerFunc(i.GetOrganizationTree))).Methods("GET")
```
Comment (robvanmieghem) [inline identityservice/organization/organizations_if.go]: duplicate of line 94

File: identityservice/organization/organizations_if.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ func OrganizationsInterfaceRoutes(r *mux.Router, i OrganizationsInterface) {
 	r.Handle("/organizations/{globalid}/apikeys/{label}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.GetAPIKey))).Methods("GET")
 	r.Handle("/organizations/{globalid}/apikeys/{label}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.UpdateAPIKey))).Methods("PUT")
 	r.Handle("/organizations/{globalid}/apikeys/{label}", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:owner"}).Handler).Then(http.HandlerFunc(i.DeleteAPIKey))).Methods("DELETE")
-	r.Handle("/organizations/{globalid}/tree", alice.New(newOauth2oauth_2_0Middleware([]string{}).Handler).Then(http.HandlerFunc(i.GetOrganizationTree))).Methods("GET")
+	r.Handle("/organizations/{globalid}/tree", alice.New(newOauth2oauth_2_0Middleware([]string{"organization:member", "organization:owner"}).Handler).Then(http.HandlerFunc(i.GetOrganizationTree))).Methods("GET")
```
Comment (robvanmieghem) [inline identityservice/organization/organizations_if.go]: @lucasvanhalst can you also make sure these scopes are added in the raml definition?

**Comment by robvanmieghem** [comment]
@lucasvanhalst Can you update the raml file accordingly?

**Comment by lucasvanhalst** [comment]
@robvanmieghem done

**Pull request merged**